demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -144,13 +144,40 @@ class TestGenericFunctions:
             ),
             tools.get_yaml,
         ),
+        (
+            os.path.join(
+                PATH_TO_HERE, "test_playbook_value_starting_with_equal_sign.yaml"
+            ),
+            tools.get_yaml,
+        ),
         (os.path.join(PATH_TO_HERE, "fake_json.json"), tools.get_json),
     ]
 
     @pytest.mark.parametrize("file_path, func", FILE_PATHS)
     def test_get_file(self, file_path, func):
         assert func(file_path)
 
+    @pytest.mark.parametrize("file_path, func", FILE_PATHS)
+    def test_get_file_or_remote_with_local(self, file_path: str, func):
+        absolute_path = Path(file_path)
+        assert tools.get_file_or_remote(absolute_path)
+        relative_path = absolute_path.relative_to(GIT_ROOT)
+        assert tools.get_file_or_remote(relative_path)
+
+    @pytest.mark.parametrize("file_path, func", FILE_PATHS)
+    def test_get_file_or_remote_with_remote(self, mocker, file_path: str, func):
```

Comment:
missing type hint for `func`

Reply:
it is not used, it's included just because I wanted to use FILE_PATHS which includes it, and other functions use it.